engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Array.java

```diff
@@ -30,6 +32,7 @@ public class Array implements TruffleObject {
    *
    * @param items the element values
    */
+  @Builtin.Method(expandVarargs = 4, description = "Creates an array with given elements.")
```

Comment:
Ooof. I think this is pointless. The `new_{n}`  family were just helpers for the java-side tests for when they had no access to stdlib. We should have got rid of them a loooong time ago. And it seems like the whole `expandVarargs` functionality is basically useless otherwise? At least I can't think of another case.